README.md

```diff
@@ -102,22 +102,68 @@ If you are connected to the Raspberry Pi via SSH, then first do:
 ```
 ## Explanation of parameters in the configuration file
 
-TODO
+The configuration file stores variables used throughout the dencam 
+code.
+
+### RECORD_LENGTH
+
+Takes a positive integer value of the length each recording will be
+(in seconds). Passing a value of 300 will make each recording 5 min
+
+### DISPLAY_RESOLUTION
+
+Takes two integer values, used to set the resolution of the display 
+pages of the Dencam.
+
+### PAUSE_BEFORE_RECORD
+
+Takes in a positive integer value which will be the seconds to wait
+after the dencam has initially started until the first recording begins
+
+### POWER_OFF_DELAY
+
+Controls the amount of time the off button needs to be held down for, until 
+the Pi shuts down.
+
+### CAMERA_RESOLUTION
+
+Takes in two positive integers that will be used to dictate the resolution
+of the camera
+
+### VIDEO_QUALITY
+
+Takes in a positive integer in the range from 1-40, 1 being the highest, 
+40 being the lowest
+
+### FRAME_RATE
+
+Controls the frame rate of the recordings, taking in a postive integer 
+from the range 1-60. The higher the number the smoother the recording, 
+but the quality of the recording reduces.We found through testing that 30 
+is a decent halfway point, where the recording is smooth enough and the 
+change in quality is not as noticable.
+
+### CAMERA_ROTATION
+
+Controls the rotation of the Camera display by taking in a positve integere representing the
+oreintation in degrees. To have the camera display in landscape, pass values 0 or 180.
+
 
 ## Using the DenCam user interface
 
 The DenCam user interface is through the PiTFT screen.  It does not
 use the touchscreen functionality of the screen: the screen itself is
 only used for display and the top two physical buttons beside the
-screen are used for control.  DenCam will start with the screen blank.
+screen are used for control.  DenCam will start on the Recording Page.
```

Comment:
Is that true? It is intended that DenCam start with the screen blank and the screen backlight off so that on all automatic resets it starts in the least power-using mode.